coreutils/yes.rs

```diff
@@ -37,29 +33,23 @@ use libc::FILE;
 //usage:#define yes_full_usage "\n\n"
 //usage:       "Repeatedly output a line with STRING, or 'y'"
 #[no_mangle]
-pub unsafe extern "C" fn yes_main(
-  mut _argc: libc::c_int,
-  mut argv: *mut *mut libc::c_char,
-) -> libc::c_int {
-  let mut pp: *mut *mut libc::c_char = 0 as *mut *mut libc::c_char;
-  let ref mut fresh0 = *argv.offset(0);
-  *fresh0 = b"y\x00" as *const u8 as *const libc::c_char as *mut libc::c_char;
-  if !(*argv.offset(1)).is_null() {
-    argv = argv.offset(1)
-  }
+pub unsafe extern "C" fn yes_main(mut argc: c_int, mut argv: *const *const c_char) -> c_int {
```

Comment:
Not important for this PR, but at some point we should figure out a way to run native rust applets without raw pointers.